bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -802,9 +809,16 @@ func (s *ksServer) SaveAppToRepo(project string, email string, repoDir string) e
 	if err != nil {
 		return err
 	}
-
-	return runCmd(fmt.Sprintf("git config user.email '%s'; git config user.name 'auto-commit'; git add .; "+
-		"git commit -m 'auto commit from deployment'; git pull --rebase; git push origin master", email))
+	if err = runCmd(fmt.Sprintf("git config user.email '%s'; git config user.name 'auto-commit'", email)); err != nil {
```

Comment:
Another way to do this would be

```golang
	cmds := []string{
		fmt.Sprintf("git config user.email %s", email),
		"git config user.name auto-commit",
		"git add .",
		`git commit "commit name"`,
		"git pull --rebase",
		"git push -f origin master",
	}
	for _, cmd := range cmds {
		if err := runCmd(cmd); err != nil {
			return err
		}
	}
```

Then you can drop the `sh -c` from the runCmd